test/moneyBack.js

```diff
@@ -0,0 +1,173 @@
+const Reverter = require('./helpers/reverter');
+const Asserts = require('./helpers/asserts');
+const MoneyBack = artifacts.require('./MoneyBack.sol');
+require('assert');
+
+contract('MoneyBack', function(accounts) {
+  const reverter = new Reverter(web3);
+  afterEach('revert', reverter.revert);
+
+  const asserts = Asserts(assert);
+  const OWNER = accounts[0];
+  
+  let moneyBack;
+
+  before('setup', () => {
+    return MoneyBack.deployed()
+    .then(instance => moneyBack = instance)
+    .then(reverter.snapshot);
+  });
+
+  it('should allow to borrow', () => {
+    const borrower = accounts[4];
+    const value = 500;
+    return Promise.resolve()
+    .then(() => moneyBack.borrow(value, {from: borrower}))
+    .then(() => moneyBack.debt(borrower))
+    .then(result => assert.equal(result.valueOf(), value))
+  });
+  
+  it('should allow to borrow from different users simultaniously', () =>{
+    const borrower1 = accounts[4];
+    const borrower2 = accounts[5];
+    const value = 500;
+    return Promise.resolve()
+    .then(() => moneyBack.borrow(value, {from: borrower1}))
+    .then(() => moneyBack.borrow(value, {from: borrower2}))
+    .then(() => moneyBack.debt(borrower1))
+    .then(result => assert.equal(result.valueOf(), value))
+    .then(() => moneyBack.debt(borrower2))
+    .then(result => assert.equal(result.valueOf(), value))
+  });
+
+  it('should allow to pay back', () => {
+    const borrower = accounts[4];
+    const value = 500;
+    return Promise.resolve()
+    .then(() => moneyBack.borrow(value, {from: borrower}))
+    .then(() => moneyBack.payback(borrower, value, {from: OWNER}))
+    .then(() => moneyBack.debt(borrower))
+    .then(result => assert.equal(result.valueOf(), 0))    
+  });
+  
```

Comment:
should payback properly when borrow different amount of money from several accounts